astropy/io/votable/tests/table_test.py

```diff
@@ -175,6 +179,80 @@ def test_write_with_format():
 def test_empty_table():
     votable = parse(
         get_pkg_data_filename('data/empty_table.xml'),
-        pedantic=False)
+        verify='warn')
     table = votable.get_first_table()
     astropy_table = table.to_table()  # noqa
+
+
+def test_verify_options(tmpdir):
```

Comment:
I feel like this can be turned into a test class with the different tests running separately as methods. This way, when one of them fail, it will be clearer *which one* is failing.

Reply:
Will do!

Reply:
Done